src/components/card/card.test.tsx

```diff
@@ -138,6 +138,19 @@ describe('EuiCard', () => {
         component.find('button').simulate('click');
         expect(handler.mock.calls.length).toEqual(1);
       });
+
+      it('should only call onClick once when title is a React node', () => {
+        const handler = jest.fn();
+        const component = mount(
+          <EuiCard
+            title={<span>Hoi</span>}
+            description="There"
+            onClick={handler}
+          />
+        );
+        component.find('button').simulate('click');
+        expect(handler.mock.calls.length).toEqual(1);
+      });
```

Comment:
This doesn't actually test the fix you added, since you're simulating a `click` event on the title `button` and not on the internal `span` (which means it matches the `link !== e.target` condition and not the new `.contains()` condition). You want something like this:

```suggestion
      it('should only call onClick once when title is a React node', () => {
        const handler = jest.fn();
        const component = mount(
          <EuiCard
            title={<span data-test-subj="click">Hoi</span>}
            description="There"
            onClick={handler}
          />
        );
        component.find('[data-test-subj="click"]').simulate('click');
        expect(handler.mock.calls.length).toEqual(1);
      });
```

You can test this for yourself - if you remove the `&& !link.contains(e.target)` logic, your current test will still pass (when it shouldn't), but if you change it to target the span, it then correctly fails as having received 2 calls.

Reply:
Everything else looks great! Just this quick test change and this PR is GTG!

Reply:
Thank you for the suggestion. I updated the test and removed the `&& !link.contains(e.target)` and I'm able to see the failure (which confirms that this test is what I needed).